### PR TITLE
zh-CN:translate tutorials/distribute/keras.ipynb

### PR DESCRIPTION
**Pull request by ccw1996**
translate https://github.com/tensorflow/docs/blob/master/site/en/tutorials/distribute/keras.ipynb

### CODE REVIEW

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+    "colab_type": "text",
+    "id": "r6P32iYYV27b"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://tensorflow.google.cn/tutorials/distribute/keras\"><img src=\"https://tensorflow.google.cn/images/tf_logo_32px.png\" />在 tensorflow.google.cn 上查看</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/distribute/keras.ipynb\"><img src=\"https://tensorflow.google.cn/images/colab_logo_32px.png\" />在 Google Colab 运行</a>\n",
```
Comment (lamberta) [inline site/zh-cn/tutorials/distribute/keras.ipynb]: remove /beta

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 2]
```diff
@@ -0,0 +1,745 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/distribute/keras.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 Github 上查看源代码</a>\n",
```
Comment (lamberta) [inline site/zh-cn/tutorials/distribute/keras.ipynb]: remove /beta

File: site/zh-cn/tutorials/distribute/keras.ipynb (hunk 1 above)
Comment (ccw1996) [reply site/zh-cn/tutorials/distribute/keras.ipynb]: the problem of ”remove /beta“ is fixed.
but   the community translation disclaimer note is not translate now. The Official Chinese translation is not available. I will try to ask people or wait for the official version

**Comment by ccw1996** [comment]
> Great, thanks!
> Can you please add a text cell that contains the community translation disclaimer note? See the top of this notebook for the text: https://github.com/tensorflow/docs/blob/master/site/zh-cn/tutorials/quickstart/beginner.ipynb

@lamberta  It's fixed. and duan says it doesn't need translate.

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "xHxb-dlhMIzW"
+   },
+   "source": [
+    "## 概述\n",
+    "\n",
+    "`tf.distribute.Strategy` API 提供了一个抽象的 API ，用于跨多个处理单元（processing units）分布式训练。它的目的是允许用户使用现有模型和训练代码，只需要很少的修改，就可以启用分布式训练。\n",
+    "\n",
+    "本教程使用 `tf.distribute.MirroredStrategy`，这是在一台计算机上的多GPU（单机多卡）进行同时训练的图形内复制（in-graph replication）。事实上，它会将所有模型的变量复制到每个处理器上，然后，通过使用 [all-reduce](http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/) 去整合所有处理器的梯度（gradients），并将整合的结果应用于所有副本之中。\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "本教程使用 `tf.distribute.MirroredStrategy`，这是在一台计算机上的多 GPU（单机多卡）进行同时训练的图形内复制（in-graph replication）。事实上，它会将所有模型的变量复制到每个处理器上，然后，通过使用 [all-reduce](http://mpitutorial.com/tutorials/mpi-reduce-and-allreduce/) 去整合所有处理器的梯度（gradients），并将整合的结果应用于所有副本之中。\n",
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 4]
```diff
@@ -0,0 +1,757 @@
...
+    "\n",
+    "`MirroredStategy` 是tensorflow中可用的几种分发策略之一。 您可以在 [分发策略指南](../../guide/distribute_strategy.ipynb) 中阅读更多分发策略。\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "`MirroredStategy` 是 tensorflow 中可用的几种分发策略之一。 您可以在 [分发策略指南](../../guide/distribute_strategy.ipynb) 中阅读更多分发策略。\n",
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 5]
```diff
@@ -0,0 +1,757 @@
...
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "r8S3ublR7Ay8"
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "# 导入 TensorFlow and TensorFlow Datasets\n",
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "# 导入 TensorFlow 和 TensorFlow 数据集\n",
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 6]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "OtnnUwvmB3X5"
+   },
+   "source": [
+    "下载MNIST数据集并从 [TensorFlow Datasets](https://tensorflow.google.cn/datasets) 加载。 这会返回 `tf.data` 格式的数据集。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "下载 MNIST 数据集并从 [TensorFlow Datasets](https://tensorflow.google.cn/datasets) 加载。 这会返回 `tf.data` 格式的数据集。"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 7]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lHAPqG8MtS8M"
+   },
+   "source": [
+    "将 `with_info` 设置为 `True` 会包含整个数据集的元数据,其中这些数据集将保存在`info`中。 除此之外，该元数据对象包括训练和测试示例的数量。 \n"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "将 `with_info` 设置为 `True` 会包含整个数据集的元数据,其中这些数据集将保存在 `info` 中。 除此之外，该元数据对象包括训练和测试示例的数量。 \n"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 8]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "psozqcuptXhK"
+   },
+   "source": [
+    "在训练具有多个GPU的模型时，您可以通过增加批量大小（batch size）来有效地使用额外的计算能力。通常来说，使用适合GPU内存的最大批量大小（batch size），并相应地调整学习速率。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "在训练具有多个 GPU 的模型时，您可以通过增加批量大小（batch size）来有效地使用额外的计算能力。通常来说，使用适合 GPU 内存的最大批量大小（batch size），并相应地调整学习速率。"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 9]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "0Wm5rsL2KoDF"
+   },
+   "source": [
+    "0-255的像素值， [必须标准化到0-1范围](https://en.wikipedia.org/wiki/Feature_scaling)。在函数中定义标准化。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "0-255 的像素值， [必须标准化到 0-1 范围](https://en.wikipedia.org/wiki/Feature_scaling)。在函数中定义标准化。"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 10]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "WZCa5RLc5A91"
+   },
+   "source": [
+    "将此功能应用于训练和测试数据，随机打乱训练数据，并[批量训练](https://tensorflow.google.cn/api_docs/python/tf/data/Dataset#batch). 请注意，我们还保留了训练数据的内存缓存以提高性能。\n"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "将此功能应用于训练和测试数据，随机打乱训练数据，并[批量训练](https://tensorflow.google.cn/api_docs/python/tf/data/Dataset#batch)。请注意，我们还保留了训练数据的内存缓存以提高性能。\n"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 11]
```diff
@@ -0,0 +1,757 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1BnQYQTpB3YA"
+   },
+   "source": [
+    "在`strategy.scope`的上下文中创建和编译Keras模型。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "在`strategy.scope`的上下文中创建和编译 Keras 模型。"
```

File: site/zh-cn/tutorials/distribute/keras.ipynb [hunk 12]
```diff
@@ -0,0 +1,757 @@
...
+    "colab_type": "text",
+    "id": "8uNqWRdDMl5S"
+   },
+   "source": [
+    "## 下一步\n",
+    "\n",
+    "* 阅读[分布式策略指南](../../guide/distribute_strategy.ipynb)。\n",
+    "* 阅读[自定义训练的分布式训练](training_loops.ipynb)教程。\n",
+    "\n",
+    "注意：`tf.distribute.Strategy`正在积极开发中，我们将在不久的将来添加更多示例和教程。欢迎您进行尝试。我们欢迎您通过[GitHub上的issue](https://github.com/tensorflow/tensorflow/issues/new) 提供反馈。"
```
Comment (Mr-Linus) [inline site/zh-cn/tutorials/distribute/keras.ipynb]:
```suggestion
    "注意：`tf.distribute.Strategy` 正在积极开发中，我们将在不久的将来添加更多示例和教程。欢迎您进行尝试。我们欢迎您通过[ GitHub 上的 issue ](https://github.com/tensorflow/tensorflow/issues/new) 提供反馈。"
```

**Comment by ccw1996** [comment]
@Mr-Linus 指出的问题都已经解决了

**Pull request merged**